test/pom.xml

```diff
@@ -12,6 +12,12 @@
     <artifactId>reactivewizard-test</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>se.fortnox.reactivewizard</groupId>
+            <artifactId>reactivewizard-jaxrs-api</artifactId>
```

Comment:
Create a new package reactivewizard-jaxrs-api-test maybe to get around the cyclic dependency issue?